tools/constraints-trusty.txt

```diff
@@ -1,3 +1,5 @@
+# pyyaml dropped python 3.4 support Nov 2019, pin to trusty version
```

Comment:
I don't think we need a specific explanatory comment; we should probably have been constraining its version all along, for our testing to be closer to the target system:

```suggestion
```

We should also constrain all of the other releases, for the same reason, but that can be a separate PR.